Compute/src/V1/Gapic/AcceleratorTypesGapicClient.php

```diff
@@ -379,71 +378,72 @@ public function get($acceleratorType, $project, $zone, array $optionalArgs = [])
+     *           Sorts list results by a certain order. By default, results are returned in alphanumerical order based on the resource name.
      *
-     *          You can also sort results in descending order based on the creation timestamp using `orderBy="creationTimestamp desc"`. This sorts results based on the `creationTimestamp` field in reverse chronological order (newest result first). Use this to sort resources like operations so that the newest operation is returned first.
+     *           You can also sort results in descending order based on the creation timestamp using `orderBy="creationTimestamp desc"`. This sorts results based on the `creationTimestamp` field in reverse chronological order (newest result first). Use this to sort resources like operations so that the newest operation is returned first.
      *
-     *          Currently, only sorting by `name` or `creationTimestamp desc` is supported.
+     *           Currently, only sorting by `name` or `creationTimestamp desc` is supported.
      *     @type string $pageToken
-     *          A page token is used to specify a page of values to be returned.
-     *          If no page token is specified (the default), the first page
-     *          of values will be returned. Any page token used here must have
-     *          been generated by a previous call to the API.
+     *           A page token is used to specify a page of values to be returned.
+     *           If no page token is specified (the default), the first page
+     *           of values will be returned. Any page token used here must have
+     *           been generated by a previous call to the API.
      *     @type bool $returnPartialSuccess
-     *          Opt-in for partial success behavior which provides partial results in case of failure. The default value is false and the logic is the same as today.
+     *           Opt-in for partial success behavior which provides partial results in case of failure. The default value is false and the logic is the same as today.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
      * @return \Google\ApiCore\PagedListResponse
      *
      * @throws ApiException if the remote call fails
-     * @experimental
      */
-    public function list_($project, $zone, array $optionalArgs = [])
+    public function list($project, $zone, array $optionalArgs = [])
```

Comment:
We are unfortunately not ready for this change (but we are very close!). This throws a syntax error in PHP <7.2.  I have a PR out that will allow us to use `list` without the underscore, but it won't be ready by tomorrow.

Reply:
Done, and [fixed](https://github.com/googleapis/gapic-generator-php/pull/285) in the microgenerator.